src/sentry/utils/db.py

```diff
@@ -11,7 +11,12 @@
 
 from django.conf import settings
 from django.db import connections, DEFAULT_DB_ALIAS
-from django.db.models.fields.related import SingleRelatedObjectDescriptor
+
+# TODO: (Django 1.9) Remove once on Django 1.9+
+try:
+    from django.db.models.fields.related_descriptors import ForwardManyToOneDescriptor
```

Comment:
@wedamija I think you may have meant `ReverseOneToOneDescriptor`.

Reply:
Oh damn, I actually followed what these people did here: https://github.com/skorokithakis/django-annoying/commit/1b858386204cdffe4315dd9bcf7a42c7289117e4

But you're right, the docs suggest something else.